### DISCUSSION TITLE
Speaker mounts.

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#22] [Ideas]
Michael,  with your added pioneer speakers, did you just shove them in there or did you make a mount? 

I'm probably just going to hot glue them in.

**Comment by gpstar81** [comment]
> Michael,  with your added pioneer speakers, did you just shove them in there or did you make a mount? 
> 
> I'm probably just going to hot glue them in.

No mounts, it is a tight fit that they do not move at all. I put some bubble padding under them so they do not vibrate against the pack shell and each other.

One day I may think about some kind of mounting system....

**Comment by DustinGrau** [comment]
Following up to close this older post, there is now a mounting system as promised. This is included in the STL folder for the pack and offers a modular approach to fitting 2 speakers in the area where the original stock speaker once resided.
https://github.com/gpstar81/haslab-proton-pack/tree/main/stl/pack/Speaker%20Mount%20V2
